content/outcomes/application-development/pair-programming.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Pair Programming
+weight: 20
+layout: single
+team:
+  - VMware Tanzu Labs
+---
+
+Lennon and McCartney. Penn and Teller. Venus and Serena.
+
+Some of the greatest achievements in history were produced by pairs of great minds working side-by-side on the same task. Software is no exception.
+
+At VMWare Tanzu Labs, we are known for being on the cutting edge of Agile software development—some might even say extreme. One software development technique in which we believe deeply is pair programming.
+
+Pair programming helps the team produce high quality software while also helping everyone on the development team have a broader understanding of the software. There are some challenges to pair programming but we’ve proven time again across hundreds of projects that the benefits far outweigh the costs.
+
+![Pair Programming](/images/outcomes/application-development/pair-programming.jpg)
+
+## What you will learn
+In this article, you will learn to:
+
+- [Define pair programming and remote pair programming](#define-pair-programming-and-remote-pair-programming)
+- [List at least five potential benefits of pair programming](#list-at-least-five-potential-benefits-of-pair-programming)
+- [List at least five potential challenges of pair programming](#list-at-least-five-potential-challenges-of-pair-programming)
+- [Describe the mechanics of the driver/navigator style of pair programming](#describe-the-mechanics-of-the-drivernavigator-style-of-pair-programming)
+
+## Define pair programming and remote pair programming
+Pair programming is a software development technique whereby two programmers work on the same problem, at the same time, on the same computer, with the goal of producing high quality software quickly. 
+
+Regarding “on the same computer,” we do not mean two people hunched over a tiny laptop. When co-located, we recommend setting up two large monitors, two sets of keyboards, two mice, and two sets of anything else the pair needs to work effectively together, such as a laptop docking station. 
+
+If the pair is not co-located, [remote pair programming](https://tanzu.vmware.com/content/blog/5-tips-for-effective-remote-pair-programming-while-working-from-home) is a good alternative. Remote pair programming is pair programming with the help of commonly available collaboration software.
+
+## List at least five potential benefits of pair programming
```

Comment:
You listed only four benefits ;-)

Reply:
Perhaps my title is confusing. Though there are 4 sub-sections, I list 18 total benefits.